src/features/locators/locators.slice.ts

```diff
@@ -49,26 +49,23 @@ const locatorsSlice = createSlice({
       locatorsAdapter.addMany(state, payload);
     },
     changeLocatorAttributes(state, { payload }: PayloadAction<ChangeLocatorAttributesPayload>) {
-      const { type, name, locator, element_id, message, isCustomName, locatorType } = payload;
+      const { locator, element_id, locatorType, ...rest } = payload;
+
       const _locator = simpleSelectLocatorById(state, element_id);
+
       if (!_locator) return;
 
-      const { fullXpath, robulaXpath } = _locator.locator;
-      const newValue = { ..._locator, locator: { ..._locator.locator }, message, type, name, isCustomName };
+      const newValue = { ..._locator, locator: { ..._locator.locator, output: locator }, locatorType, ...rest };
```

Comment:
don't save `locator` to `output`. save it to `xPath ` or `css`

Reply:
fixed